guide/src/class.md

```diff
@@ -23,12 +23,10 @@ This chapter will discuss the functionality and configuration these attributes o
 To define a custom Python class, add the `#[pyclass]` attribute to a Rust struct or a fieldless enum.
 ```rust
 # #![allow(dead_code)]
-# use pyo3::prelude::*;
+use pyo3::prelude::*;
+
 #[pyclass]
-struct MyClass {
-    # #[pyo3(get)]
-    num: i32,
-}
+struct Number(i32);
```

Comment:
This is the first `#[pyclass]` example most users will see. I tend to think structs with named fields are more typical than tuple structs (although tuple structs definitely have their place). 

Maybe have examples of all three - named fields, tuple struct, and enum?

Then the rest of the examples can proceed to use the tuple struct.

Reply:
Sure, I'll show all forms 👍